<launch>
  <node name="calibrate" pkg="colab_reachy_grasp" type="calibrate.py" respawn="false" output="screen">
    <param name="side" value="left" />
    <param name="save_file_path" value="~/Documents/ReachyLeftArmErrorMap.txt" />
    <param name="april_tag_topic" value="cubePose" />
    <param name="min_x" value="0.3" />
    <param name="max_x" value="0.7" />
    <param name="min_y" value="0.0" />
    <param name="max_y" value="0.4" />
    <param name="min_z" value="0.7" />
    <param name="max_z" value="1.1" />
    <param name="div_x" value="4" />
    <param name="div_y" value="4" />
    <param name="div_z" value="4" />

    <!--
    <param name="min_x" value="337.5" />
    <param name="max_x" value="637.5" />
    <param name="min_y" value="-77.5" />
    <param name="max_y" value="827.5" />
    <param name="min_z" value="-37.5" />
    <param name="max_z" value="-637.5" />
    <param name="div_x" value="30" />
    <param name="div_y" value="89" />
    <param name="div_z" value="60" />
    -->

  </node>
</launch>
